package.json

```diff
@@ -37,12 +37,11 @@
     "babel-cli": "6.9.0",
     "babel-preset-es2015": "6.9.0",
     "babel-preset-stage-0": "6.5.0",
+    "babel-register": "^6.9.0",
```

Comment:
[`6.9.0` is the latest version](https://www.npmjs.com/package/babel-register). I tried installing `cashay` from scratch and it failed because of `babel-register` being specified as `6.9.1` which doesn't exist.